code/modules/antagonists/eldritch_cult/knowledge/rust_lore.dm

```diff
@@ -156,9 +187,7 @@
 	human_user.adjustBruteLoss(-4, FALSE)
 	human_user.adjustFireLoss(-4, FALSE)
 	human_user.adjustToxLoss(-4, FALSE, forced = TRUE)
-	human_user.adjustOxyLoss(-2, FALSE)
-	human_user.adjustStaminaLoss(-20)
-	human_user.AdjustAllImmobility(-10)
+	human_user.adjustOxyLoss(-4, FALSE)
```

Comment:
I'd suggest keeping the stamina regen and stun recovery parts of this, just in case you get batonned or knocked down and manage to crawl onto a rust tile

Reply:
good idea